test/extended/cli/debug.go

```diff
@@ -278,4 +280,83 @@ spec:
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(out).To(o.ContainSubstring("image: quay.io/wildfly/wildfly-centos7"))
 	})
+
+	g.It("ensure that the label is set for node debug", func() {
+		var err error
+
+		ns := ocNodeClient.Namespace()
+
+		err = ocNodeClient.AsAdmin().Run("label").Args("namespace", ns, "pod-security.kubernetes.io/enforce=privileged", "pod-security.kubernetes.io/audit=privileged", "--overwrite").Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		err = ocNodeClient.AsAdmin().Run("auth").Args("can-i", "get", "nodes").Execute()
+		o.Expect(err).NotTo(o.HaveOccurred(), "User should be able to get nodes")
+
+		nodesOutput, err := ocNodeClient.AsAdmin().Run("get").Args("nodes", "-l", "node-role.kubernetes.io/worker=", "--no-headers", "-o", "custom-columns=NAME:.metadata.name,STATUS:.status.conditions[?(@.type=='Ready')].status").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(nodesOutput).NotTo(o.BeEmpty(), "No worker nodes found")
+
+		var readyWorkerNode string
+		for _, line := range strings.Split(strings.TrimSpace(nodesOutput), "\n") {
+			fields := strings.Fields(line)
+			if len(fields) >= 2 && fields[1] == "True" {
+				readyWorkerNode = fields[0]
+				break
+			}
+		}
+		o.Expect(readyWorkerNode).NotTo(o.BeEmpty(), "No ready worker node found")
+
+		done := make(chan error, 1)
+		go func() {
```

Comment:
Any failure in these tests bring about a release blocker bug that needs to be investigated. Running some test cases in goroutines make investigation harder. I don't think, we need goroutines. You can run debug node by passing this flag https://github.com/openshift/oc/blob/0f09f6e9cc7fdaab8942ad118d0717a6d527746b/pkg/cli/debug/debug.go#L264.